style.css

```diff
@@ -0,0 +1,10 @@
+
+#board {
```

Comment:
Why FFS we have anything about board here. Task is about tile. And I didn't event put any styling in definition of done and acceptance criteria anyway